lib/app.js

```diff
@@ -167,7 +167,7 @@ const components = {
    * @returns A string with the custom label to display for the given leg, or null to render no label.
    */
   getTransitiveRouteLabel: (itineraryLeg) => {
-    return itineraryLeg.routeShortName // null or undefined or empty string will tell transitive-js to not render a route label.
+    return itineraryLeg.routeShortName || itineraryLeg.route.shortName // null or undefined or empty string will tell transitive-js to not render a route label.
```

Comment:
Do we know `route` will exist? Will it always have a shortname?

Reply:
I added a conditional to `route`. If there's no short name, it'll return `undefined` which in theory transitive should handle gracefully ( it doesn't but that's out of the scope of this PR)